### DISCUSSION TITLE
hekili to the next step ?

### DISCUSSION THREAD

**Discussion by Saephirae** [Hekili/hekili#1284] [Ideas]
hi ! 

loving hekili, reading some many comments i had an idea : 

Would it be possible to allow hekili to read some weakaura  to allow hekili to tell when to counterspell or stun !
So it would help hekili "having a judgment" on what behaviour having in CC management.

was having a tought of a new tab that would display  opportunity to use the proper cooldown ( stun, counterspell for exemple)  and when targetting the proper npc  imputting the spell into hekili spell line .

just an idea  with no knowledge of possibility or how hard on work time  it would be  no judgment please ^^

Thanks again for your work !

**Comment by Hekili** [comment]
Pretty much every specialization that has an interrupt has usage of the interrupt built into the addon already.  They are toggled off by default (see /hekili > Toggles > Show Interrupts).

There's no way to determine if an enemy can be stunned, so I generally don't include those in the addon.
